src/views/Auth.jsx

```diff
@@ -1,5 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import bookIMG from '../assets/a66849ac2514905d94da.png';
+import { useUser } from '../context/UserContext';
 
 export default function Auth() {
-  return <div>Auth</div>;
+  const [signingIn, setSigningIn] = useState(true);
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const history = useHistory();
+  const { setUser, user, handleAuth } = useUser();
+
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    try {
+      const resp = await handleAuth(email, password, signingIn);
+      setUser(resp);
+      history.push('/');
+    } catch (error) {
+      setError(error.message);
+    }
+  };
+  return (
+    <div>
+      <img src={bookIMG} alt="Book" />
+      <h1>{signingIn ? 'Sign in to your account' : 'Create an account'}</h1>
```

Comment:
It's difficult to know how to sign up with a new account. When I tried to sign in with a user that didn't exist it just rendered a blank page. Adding some user messages or redirecting to creating an account would provide a better user experience.